lib/gemfury/command/app.rb

```diff
@@ -221,12 +224,22 @@ def push_files(command, gem_paths)
       die!("Problem: No valid packages found", nil, command)
     end
 
+    push_options = { }
+    if options[:public]
```

Comment:
We want to support explicitly setting `public=false`. The benefit there is if you're explicitly setting `public` to false for an upload, the API will throw an error if the package has been previously set to public in the account. This prevents accidentally making a new upload public based on existing package, if you're setting it explicitly to `public=false`

So I guess this should be:
```suggestion
    unless options[:public].nil?
      push_options[:public] = options[:public]
    end
```

Not sure if you need to call `to_s` on `options[:public]` here or if Faraday will correctly marshal it into string.